### PR TITLE
SDPA-456 Blocked and then repeat publishing caused by "Failed to move to archived." error

### PR DESCRIPTION
**Pull request by mugurrus**
code profiling functionality

### CODE REVIEW

File: superdesk/factory/default_settings.py [hunk 1]
```diff
@@ -415,3 +416,7 @@ def env(variable, fallback_value=None):
 
 # max transmit items to be fetched from mongo
 MAX_TRANSMIT_QUERY_LIMIT = int(env('MAX_TRANSMIT_QUERY_LIMIT', 500))
+
+# Code profiling for performance analysis
+ENABLE_PROFILING = True
```
Comment (petrjasek) [inline superdesk/factory/default_settings.py]: imo this should be disabled by default

**Comment by vied12** [comment]
:+1:  a little note about how to use it could help me to understand

**Comment by vied12** [comment]
Never mind, there is an UI. I start to understand now :)

**Comment by petrjasek** [comment]
would this also profile workers? or is this supposed to work only for rest api processes

**Comment by mugurrus** [comment]
@petrjasek Profiling for workers implemented with the last commit. For now I added profiling for publish:enqueue and publish:transmit tasks but other can be added easily.

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**